src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,8 @@
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
```

Comment:
I feel like we should change this to INDEX_FIRST and use for everything. But Syinyi's code and my code also need. So maybe u leave it as INDEX_FIRST_PERSON, remove the transaction indexes, then once all our code merge, we can rename everything all at once to INDEX_FIRST?

Reply:
Yeah! That was what I thought as well! Will it be okay for me to use INDEX_FIRST_PERSON though? Cause mine is technically a Transaction not Person, xD Will they mark us down.